apps/charterafrica/src/pages/index.page.js

```diff
@@ -42,6 +42,47 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
     fallbackLocale: defaultLocale,
   });
 
+  const { docs: homePage } = await payload.findPage("home", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  const { blocks: pageBlocks } = homePage[0] ?? {
+    blocks: [],
+  };
+
+  const blocks =
+    pageBlocks?.map(({ blockType, ...other }) => ({
+      ...other,
+      slug: blockType,
+    })) || [];
+
+  const spotlight = blocks.find((block) => block.slug === "spotlight");
+
+  const spotlightItems = spotlight?.items?.map((item) => {
+    const { item: itemData, ...rest } = item;
+    return {
+      ...rest,
+      item: {
+        ...itemData,
+        image: {
+          src: itemData.image.url,
+          alt: itemData.image.alt,
+        },
+        date: new Date(itemData.date).toLocaleDateString(locale, {
+          year: "numeric",
+          month: "short",
+          day: "numeric",
+        }),
+        link: {
+          href: itemData?.link.href || "#", // TODO: handle reference links
+        },
+      },
+    };
+  });
+
+  spotlight.items = spotlightItems;
```

Comment:
```suggestion
  spotlight.items = spotlightItems || null;
```

`undefined` can't be serialized.